app/javascript/dashboard/store/modules/helpCenterPortals/actions.js

```diff
@@ -0,0 +1,90 @@
+import HelpCenterPortalsAPI from 'dashboard/api/helpCenter/portals.js';
+
+export const actions = {
+  fetchAllHelpCenters: async ({ commit }) => {
+    try {
+      commit('setUIFlag', { isFetching: true });
+      const { data } = await HelpCenterPortalsAPI.get();
+      data.forEach(helpCenter => {
+        const { id: helpCenterId } = helpCenter;
+
+        commit('addHelpCenterEntry', helpCenter);
```

Comment:
Let us not use the term help center. This is Portals API. Let us rename the variable to Portals.